### PR TITLE
Fix the issue with missing images in the Reality Capture data.

### PR DESCRIPTION
**Pull request by dkorolov**
I had practical situations when exported data from Reality Capture was not consistent. The number of cameras exported in CSV file was higher than the number of images on a very big dataset. This can be related to Reality Capture project settings and export options. CSV file and source images were exported separately. 
This situation gives an error with ns-process-data
These changes fix the issue with missing images in the Reality Capture data.

### CODE REVIEW

**Comment by machenmusik** [comment]
Interesting... which CSV export option are you using from RealityCapture?
I had success with "Internal\External camera parameters" -- albeit with modest image sets.

**Comment by dkorolov** [comment]
I experiment with the big private dataset. I tried to use export CSV with "Internal\External camera parameters" and use "Original Images with Image list" to get source images in one folder. For some reason were exported fewer images than cameras in CSV  (May be some RealityCapture project settings caused this, not sure). Also, I remove part of the images that did not fit the main resolution. The original dataset was created for high-quality photogrammetry, the project contains about 2500 cameras. Source images for RealityCapture were distributed in several folders.  So I try to consolidate it using the export "Original Images with Image list"

File: nerfstudio/process_data/realitycapture_utils.py [hunk 1]
```diff
@@ -58,15 +59,30 @@ def realitycapture_to_json(
             for column, value in row.items():
                 cameras.setdefault(column, []).append(value)
 
-    img = np.array(Image.open(output_dir / image_filename_map[cameras["#name"][0].split(".")[0]]))
+    # check the first exist image with camera data
```
Comment (tancik) [inline nerfstudio/process_data/realitycapture_utils.py:62]: Maybe remove, or make more specific

**Review by tancik — APPROVED** [review]
LGTM

File: nerfstudio/process_data/realitycapture_utils.py (hunk 1 above)
Comment (dkorolov) [reply nerfstudio/process_data/realitycapture_utils.py]: I removed comment

**Pull request merged**